app/assets/stylesheets/filter.scss

```diff
@@ -0,0 +1,117 @@
+.slider-nav {
+    display: flex;
+    align-items: center;
+}
+
+.nav-price {
+    font-family: $font;
```

Comment:
Why don't you declare your font family globally? You shouldn't need to add it for all the elements